### ISSUE TITLE
Problem when reading documents multiple times

### ISSUE THREAD

**Issue by RichezA** [mganss/ExcelMapper#74]
Hello,

I'm using your library (which i find very useful and straightforward !!). I did put my "Excel-To-Core objects" part in a separate module to use it in another project if i had to.

The problem i encountered is that when i attempt to re-parse the data another time (after an insertion failure in my database or a user request), I get a NullReferenceException:
`object reference not set to an instance of an object` with the following stack trace:
`   at NPOI.Util.LocaleUtil.GetLocaleCalendar(TimeZone timeZone)
   at NPOI.Util.LocaleUtil.GetLocaleCalendar()
   at NPOI.SS.UserModel.DateUtil.GetJavaCalendar(Double date, Boolean use1904windowing, TimeZone timeZone, Boolean roundSeconds)
   at NPOI.SS.UserModel.DateUtil.GetJavaDate(Double date, Boolean use1904windowing)
   at NPOI.SS.UserModel.DateUtil.GetJavaDate(Double date)
   at Ganss.Excel.ExcelMapper.GetCellValue(ICell cell, ColumnInfo targetColumn)
   at Ganss.Excel.ExcelMapper.<Fetch>d__58.MoveNext()
   at System.Linq.Enumerable.<OfTypeIterator>d__61`1.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at `

I don't know how to fix this issue. I can provide more informations if needed.
Have a nice day,

**Comment by mganss** [comment]
This looks like #51. Can you post the Excel file and the code used to map and read the file?

**Comment by RichezA** [comment]
Hello,

Sorry for the late response. I checked #51 and it seems to be the same problem. Accessing twice DateTime fields. Here is a sample of the excel file:
[SampleExcel.xlsx](https://github.com/mganss/ExcelMapper/files/5259990/SampleExcel.xlsx)

In the program, I first fetch columns from A to AU. And I fetch this file a second time for columns AK to AN.
The problem seems to come from the fact that there also is date in the AK column.

```cs
        private async Task<List<Models.Sample1>> FetchSample1(Stream file)
        {
            file.Position = 0;
            this.excelMapper.MinRowNumber = 2;
            return (await this.excelMapper.FetchAsync<Models.Sample1>(file, "Base"))
                .Where(obj => obj.Code != null && obj.Name != null && obj.Id != null)
                .ToList();
        }


        private async Task<List<Models.Sample2>> FetchWaterCaptationComplementsAsync(Stream file)
        {
            file.Position = 0;
            this.excelMapper.HeaderRow = false;
            this.excelMapper.MinRowNumber = 2;
            var samples = (await this.excelMapper.FetchAsync<Models.Sample2>(file, "Base")).ToList();
            this.excelMapper.HeaderRow = true;
            return samples;
        }
```

This is the code I use to fetch the data

**Comment by RichezA** [comment]
The problem seems to be even more annoying now. Then I had to restart my program. Now I have to restart my whole computer in order to test it a second time.

**Comment by mganss** [comment]
Can you post the class definitions for `Sample1` and `Sample2`?
Are you using a single `ExcelMapper` object for both calls?

**Comment by RichezA** [comment]
Hi, those are my class definitions:
Sample1
```cs
public class Sample1
{
    [Column(1)]
    public string Identifier {get;set;}

    [Column(2)]
    public string Code {get;set;}

    [Column(3)]
    public string Name {get;set;}

    [Column(4)]
    public int StatusId {get;set;}

    [Column(5)]
    public int ExternalId {get;set;}

    [Column(6)]
    public string ExternalStatus {get;set;}

    [Column(7)]
    public string Municipality {get;set;}

    [Column(8)]
    public string City {get;set;}

    [Column(9)]
    public string Place {get;set;}

    [Column(10)]
    public string ExternalCode {get;set;}

    [Column(11)]
    public string AccessStatus {get;set;}

    [Column(12)]
    public string ExternalAccessStatus {get;set;}

    [Column(13)]
    public string ExternalStatus2 {get;set;}

    [Column(14)]
    public DateTime? ReceptionDate {get;set;}

    [Column(15)]
    public DateTime? AdviceDate {get;set;}

    [Column(16)]
    public DateTime? DomiDate {get;set;}

    [Column(17)]
    public DateTime? GivenDate {get;set;}

    [Column(18)]
    public DateTime? SampleDate {get;set;}

    [Column(19)]
    public DateTime? DataDate {get;set;}

    [Column(20)]
    public DateTime? DataSentDate {get;set;}

    [Column(21)]
    public DateTime? DataReceptionDate {get;set;}

    [Column(22)]
    public DateTime? DataAdviceGivenDate {get;set;}

    [Column(23)]
    public DateTime? DataFeedbackDate {get;set;}

    [Column(24)]
    public DateTime? TestingDate {get;set;}

    [Column(25)]
    public DateTime? TestingAckDate {get;set;}

    [Column(26)]
    public DateTime? StagingDate {get;set;}

    [Column(27)]
    public DateTime? StagingAckDate {get;set;}

    [Column(28)]
    public DateTime? OverallTestingDate {get;set;}

    [Column(29)]
    public DateTime? ProdDate {get;set;}

    [Column(30)]
    public DateTime? ProdAckDate {get;set;}

    [Column(31)]
    public DateTime? DemoDate {get;set;}

    [Column(32)]
    public DateTime? MeetingDate {get;set;}

    [Column(33)]
    public DateTime? EndOfProjectDate {get;set;}

    [Column(34)]
    public string WorkingStatus {get;set;}

    [Column(35)]
    public string BridgingStatus {get;set;}

    [Column(36)]
    public int Suspension {get;set;}
}
```
Sample2
```cs
public class Sample2
{
    [Column(37)]
    public DateTime? CaptDate {get;set;}

    [Column(38)]
    public string Identifier {get;set;}

    [Column(39)]
    public int Value {get;set;}

    [Column(40)]
    public int Disposed {get;set;}
}
```

I originally used the same one, I also tested with separate object instances but it didn't change anything.

**Comment by mganss** [comment]
This is due to https://github.com/tonyqus/npoi/issues/397. I have applied the workaround mentioned there until a new version of NPOI is released.

**Comment by RichezA** [comment]
Ok. Thanks for the help ! Have a nice day!

**Comment by mganss** [comment]
The workaround has been removed in 5.1.294 as the underlying cause was fixed in NPOI 2.5.3.
